utils.go

```diff
@@ -20,10 +20,6 @@ func WriteTOML(path string, data interface{}) error {
 	return toml.NewEncoder(f).Encode(data)
 }
 
-func buildpackIDToDir(id string) string {
+func escape(id string) string {
```

Comment:
The new name of this func feels a little ambiguous. I get that we need it for more than dir names now, but maybe we should still capture the intent, like `escapeBuildpackID`? Or maybe I've just been Java-ing too long.